<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSystemWidget</class>
 <widget class="QWidget" name="FileSystemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>4</number>
   </property>
   <item row="4" column="0">
    <widget class="FileSystemView" name="listView">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="upButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Up</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/tango/images/icons/tango/go-up.png</normaloff>:/tango/images/icons/tango/go-up.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="selectButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Select Folder</string>
       </property>
       <property name="icon">
        <iconset resource="../../icons.qrc">
         <normaloff>:/tango/images/icons/tango/folder-saved-search.png</normaloff>:/tango/images/icons/tango/folder-saved-search.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterEdit">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Clear Filter</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QWidget" name="buttonPanel" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="playAllButton">
        <property name="toolTip">
         <string>Play All of Current Folder</string>
        </property>
        <property name="text">
         <string>Play All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="playRootButton">
        <property name="toolTip">
         <string>Play Root of Current Folder</string>
        </property>
        <property name="text">
         <string>Play Root</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addAllButton">
        <property name="toolTip">
         <string>Add All of Current Folder</string>
        </property>
        <property name="text">
         <string>Add All</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addRootButton">
        <property name="toolTip">
         <string>Add Root of Current Folder</string>
        </property>
        <property name="text">
         <string>Add Root</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QWidget" name="progressPanel" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSystemView</class>
   <extends>QListView</extends>
   <header>widgets/fs/filesystemview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>upButton</tabstop>
  <tabstop>selectButton</tabstop>
  <tabstop>filterEdit</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>clearButton</tabstop>
  <tabstop>playAllButton</tabstop>
  <tabstop>playRootButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
